src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
```

Comment:
Personally, I don't think we should move the styles to the Preview SDK because, the additional styles are in support of the Sidebar moving to the bottom on small screens. My thinking is it's better to keep all the functionality in one repo rather than splitting it.